src/amo/components/AddonReviewListItem/index.js

```diff
@@ -231,7 +231,12 @@ export class AddonReviewListItemBase extends React.Component<Props> {
         {reviewBody}
         <div className="AddonReviewListItem-byline">
           {review && !isReply ?
-            <Rating styleName="small" rating={review.rating} readOnly />
+            <Rating
+              styleSize="small"
+              isOwner={siteUser && review && review.userId === siteUser.id}
```

Comment:
The logic of figuring out who owns the review should reside within `Rating` so that the component can be re-used more easily. Consider this: if you want to render `Rating` within another component using this `isOwner` approach, you'd have to copy/paste all the logic and tests for checking who owns the review.

Also, there is an easier way to do this -- a pattern that you might not have seen yet. The `Rating` component can map directly to the logged in user data. In fact, [`AddonReviewListItem`](https://github.com/mozilla/addons-frontend/blob/master/src/amo/components/AddonReviewListItem/index.js#L240) already does exactly what you need to do: it checks the `siteUser` against `review.user.Id`. Check out [its mapper](https://github.com/mozilla/addons-frontend/blob/master/src/amo/components/AddonReviewListItem/index.js#L316) to see how it works. With that mapper pattern, the caller won't have to pass in anything special to `Rating`.